letsdrill.rb

```diff
@@ -1,24 +1,42 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  if integer >= 90
+    return 'A'
+  elsif integer >= 80
+    return 'B'
+  elsif integer >= 70
+    return 'C'
+  elsif integer >= 60
+    return 'D'
+  end
+  return 'F' # if it gets this far it has to be an F
 
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  if array.empty?
+    return nil
+  end
+  #sets a value for the length of the smallest string to be the first
+  smallest_length = array[0].length
+  smallest = array[0];
+  for i in array
+    if i.length < smallest_length
+      smallest_length = i.length;
+      smallest = i;
+    end
+  end
+  return smallest;
```

Comment:
There is a builtin method that basically does this all for you. Scan the methods available on Enumerable objects (Arrays are Enumerable), hopefully you'll find something helpful: http://ruby-doc.org/core-2.0.0/Enumerable.html